unipipe/commands/generate.ts

```diff
@@ -42,12 +57,88 @@ function generateUuid() {
   console.log(uuid.generate());
 }
 
-function generateCatalog() {
+async function generateCatalog() {
   console.log(catalog);
```

Comment:
f: This behaves differently from the other generate commands, by printing out the catalog.

This is an example of an inconsistency that would only need a fix in one place, when using one function for all places, where we allow to choose the output of the generate command.

Reply:
the new function is alive 🕺